interfaces/Portalicious/src/app/components/form/form-default.component.html

```diff
@@ -15,6 +15,7 @@
       type="submit"
       rounded
       [loading]="mutation().isPending()"
+      [disabled]="disabled()"
```

Comment:
Currently, in Portalicious, we only disable buttons when we are in a loading state. Rather than disabling the button, I imagine Tal would prefer us to show an error when the user tries to click it.